### PR TITLE
Replotting requirements

### PR DESCRIPTION
**Pull request by dariolina**
Add info on how much a farmer has to replot if they join at genesis

### CODE REVIEW

File: docs/consensus/consensus/plotting.md [hunk 1]
```diff
@@ -55,8 +55,18 @@ As a result, a farmer has a unique encoded replica that is difficult to compress
...
+Recall that when plotting a sector, the farmer saves the history size at the time, and it determines a point in the future when the sector will expire. When a sector reaches its expiry point, the block proposer challenge solutions coming from this sector will no longer be accepted by other peers, incentivizing the farmer to update their plot. The farmer erases the expired sector and repeats the Plotting process anew, replicating a fresh history sample. Each replotting creates a new sector in memory and saves it to disk in a single write operation.
 
 <!-- ![Replotting](../../../src/Images/Replotting.png) -->
 
 In a plot spanning multiple gigabytes, the sectors will be updated randomly, one at a time, so replotting is amortized over a long period. There is never a moment when a farmer needs to erase and re-create their whole plot and miss out on challenges. The plot refreshing will be practically invisible to the farmer and allow their uninterrupted participation in consensus.
+
+The bigger the chain grows and the longer the farmer participates in the network, the less frequent the replotting on their disks will be.
```
Comment (nazar-pc) [inline docs/consensus/consensus/plotting.md:64]: I don't think the participation part is accurate here.

File: docs/consensus/consensus/plotting.md [hunk 2]
```diff
@@ -55,8 +55,18 @@ As a result, a farmer has a unique encoded replica that is difficult to compress
...
+Assuming a 1 TB SSD plot, a farmer who joined at genesis will on average need to replot 3.5 TB of data by the time the chain history reaches 1TB in size, 6.5 TB of data by the time history reaches 10 TiB and 9.2 TB by the time history reaches 50 TiB. Given a common TBW (Total Bytes Written) for consumer grade 1 TB SSDs of 300-600 TB, Subspace farming requires below 3% of the SSD's total endurance to farm over several years (for comparison, Ethereum's full chain data size is ~16 TiB as of October 2023 via [Etherscan](https://etherscan.io/chartsync/chainarchive)).
```
Comment (nazar-pc) [inline docs/consensus/consensus/plotting.md:65]: This is something you can also post on the forum here: https://forum.subspace.network/t/what-triggers-a-sector-to-start-replotting/1688

File: docs/consensus/consensus/plotting.md (hunk 1 above)
Comment (dariolina) [reply docs/consensus/consensus/plotting.md:64]: Participation part mostly refers to ROI on the plots they already created, so I still think it's relevant. But you're right the replotting trigger doesn't depend on it.

**Review by nazar-pc — APPROVED** [review]

File: docs/consensus/consensus/plotting.md (hunk 2 above)
Comment (dariolina) [reply docs/consensus/consensus/plotting.md:65]: I will post a link to this section directly

**Pull request merged**